src/components/MatPass.tsx

```
@@ -0,0 +1,478 @@
+};
+
+export default function MatPass({ parentTab = 'matpass' }: MatPassProps) {
+  const [addedToCart, setAddedToCart] = useState<{[key: string]: boolean}>({})
+  const [animatingCards, setAnimatingCards] = useState<{[key: string]: boolean}>({})
+  const { language } = useLanguageContext();
+  
+  const { data: bundleData, loading: bundleLoading, error: bundleError } = useQuery<GetBundleQuery>(GET_BUNDLE_TYPES, { 
+    variables: { contextId: "ec966559-0580-4adb-bc6b-b150c56f935c"} 
+  });
+
+  const addToCart = (id: number, name: string, price: number, isPass: boolean = false, isAcro: boolean = false, bundleTypeId?: string) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)